docker/proxy/Dockerfile

```diff
@@ -0,0 +1,17 @@
+###############################################
```

Comment:
I found it was useful to be able to hop into the container and edit the configuration files and stop/start nginx, which isn't possible from the official image. Now that the configuration has been sorted I can update the Dockerfile with the official image.